tests/cypress/integration/IBF-dashboard/timelinebutton.js

```diff
@@ -0,0 +1,27 @@
+import selectors from "../../support/selectors";
+import constants from "../../support/constants";
+
+// timeline button validation
+describe("timeline button validation", () => {

```

Comment:
I get this error when running this test,

<img width="423" alt="Screenshot 2021-07-26 at 11 03 27" src="https://user-images.githubusercontent.com/3880591/126963397-b8070700-89b0-4a3a-9139-9b98c8b39bb1.png">

Here is the stack trace,
```
    at check (webpack:///node_modules/cypress-wait-until/src/index.js:62:1)
    at resolveValue (webpack:///node_modules/cypress-wait-until/src/index.js:77:1)
    at Context.eval (webpack:///node_modules/cypress-wait-until/src/index.js:66:1)
From previous event:
    at Context.thenFn (https://ibf-test.510.global/__cypress/runner/cypress_runner.js:156453:24)
    at Context.then (https://ibf-test.510.global/__cypress/runner/cypress_runner.js:156892:22)
    at Context.<anonymous> (https://ibf-test.510.global/__cypress/runner/cypress_runner.js:171546:22)
    at <unknown> (https://ibf-test.510.global/__cypress/runner/cypress_runner.js:170970:16)
From previous event:
    at runCommand (https://ibf-test.510.global/__cypress/runner/cypress_runner.js:170949:9)
    at next (https://ibf-test.510.global/__cypress/runner/cypress_runner.js:171095:15)
From previous event:
    at next (https://ibf-test.510.global/__cypress/runner/cypress_runner.js:171095:35)
    at <unknown> (https://ibf-test.510.global/__cypress/runner/cypress_runner.js:171123:17)
```

Reply:
This component has been renamed to `Lead Time` instead of `Timeline` as mentioned in the DevOps item.

I understand the cause of confusion as it's inconsistent in the code. So it's not necessary to rename this as a part of this item. But it would be nice to keep this consistent from now on.

Reply:
changed to lead time.